e2e/scenarios/basic/env.scenario

```diff
@@ -22,8 +22,7 @@ ENDPOINT_LOGIN=/login
 ENDPOINT_METADATA=/metadata
 ENDPOINT_LOGOUT=/logout
 
-SPID_VALIDATOR_URL=http://localhost
-SPID_TESTENV_URL=https://spid-testenv2:${SPID_TESTENV_PORT}
+SPID_VALIDATOR_URL=http://spid-saml-check:${SPID_DEMO_IDP_PORT}/demo
```

Comment:
It is ok using the validator endpoint for this scenario, since it is already configured correcly in `io-spid-commons`

However for the future would be great to provide an environment variable relative to 
pagopa/io-spid-commons#111 and use that instead.

The use of an ad-hock environment variable, would allow to use spid-saml-check container for its double pourpose, both demo IdP and Validator

Reply:
I actually had a chat with @BurnedMarshal on which he told me that `io-spid-commons` calls validator what we actually call demo. This is because of legacy things both on spid-saml-check and us.

Long story short, I agree with you but we might give it an extra round of thoughts.

Reply:
"demo" is the mock of ad IdP i.e the equivalent of the defunct testenv2 (cfr [How to use it as a SPID Demo](https://github.com/italia/spid-saml-check#how-to-use-it-as-a-spid-demo))

"Valdator" is a tool to providing a set of compliance test to semantically assess SP Metadata, SAML Request and SAML Response (cfr [How to use it as a SPID Validator](https://github.com/italia/spid-saml-check#how-to-use-it-as-a-spid-validator) )

Since both are in the same container, can lead to some ambiguity... but conceptually are two separated tools requiring two separate entity ids